pkg/services/dashboards/manager/dashboard_service.go

```diff
@@ -18,6 +19,15 @@ import (
 	"github.com/grafana/grafana/pkg/util/errutil"
 )
 
+var (
+	provisionerPermissions = map[string][]string{
+		accesscontrol.ActionFoldersCreate:    {},
+		accesscontrol.ActionFoldersWrite:     {accesscontrol.ScopeFoldersAll},
+		accesscontrol.ActionDashboardsCreate: {accesscontrol.ScopeFoldersAll},
+		accesscontrol.ActionDashboardsWrite:  {accesscontrol.ScopeFoldersAll},
+	}
```

Comment:
Yes, so whenever grafana runs dashboard and folder provisioning we create a SingedInUser model. We need to set this permissions on that user so that the provisioner has access to perform these tasks